pkg/ipamd/rpc_handler_test.go

```diff
@@ -82,7 +81,7 @@ func TestServer_AddNetwork(t *testing.T) {
 
 		var expectedCIDRs []string
 		for _, cidr := range tc.vpcCIDRs {
```

Comment:
This `for` loop can be simplified to just `expectedCIDRs := tc.vpcCIDRs`

.. but wait. Isn't `expectedCIDRs` overwritten on the next line?  Maybe these few lines do nothing (other than an expensive way to pre-allocate the array capacity) and should just be removed entirely?

Reply:
Heh, yeah, they do nothing. `staticcheck` found this as well, but I didn't want to change it in #1064 because I had changed the return values in this PR to be `[]string` instead of `[]*string`.